server/events/command_runner.go

```diff
@@ -212,7 +212,7 @@ func (c *DefaultCommandRunner) checkVarFilesInPlanCommandAllowlisted(cmd *Commen
 		return nil
 	}
 
-	return c.VarFileAllowlistChecker.Check(cmd.Flags)
+	return c.VarFileAllowlistChecker.Check(cmd.ExtraArgs)
```

Comment:
Did you mean to change this? I thought you were only changing the Flags that were added for this pr and not all the references to Flags, no?

Reply:
I figured we only need the changes in api controller and api controller test 

cc @GenPage @jamengual

Reply:
my understanding was to change `events.CommentCommand.Flags -> events.CommentCommand.ExtraArgs`.  
https://github.com/runatlantis/atlantis/blob/d08912c79157011d6aa744a1993e7bd79bd4a799/server/controllers/api_controller.go#L59


The `CommentCommand` struct was used in a lot of places, hence all the file changes when I renamed the symbol.  I can switch it back if needed.

Reply:
sorry that is not what I meant, please revert it back.

Reply:
👍🏾 sounds good. reverted the last commit.